tests/eas/load_tracking.py

```diff
@@ -45,6 +45,15 @@ class _LoadTrackingBase(LisaTest):
         'modules': ['cpufreq', 'cgroups'],
     }
 
+    @memoized
+    @staticmethod
+    def _get_cpu_capacity(test_env, cpu):
+        if test_env.nrg_model:
+            return test_env.nrg_model.get_cpu_capacity(cpu)
+        else:
+            return test_env.target.read_int(
+                '/sys/devices/system/cpu/cpu{}/cpu_capacity'.format(cpu))
```

Comment:
Hmm - my thinking was that the `EnergyModel` probably read its data from the target anyway so it's a waste of time (and it can be quite a lot of time) to do that again.  If you provided it by hand then you probably had a good reason for doing that so we should respect it.